cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -113,8 +113,9 @@ func (w *wizard) run() {
 		fmt.Println(" 13. Print latest jobs")
 
 		if w.transcoder {
-			fmt.Println(" 14. Become a transcoder")
-			fmt.Println(" 15. Set transcoder config")
+			fmt.Println(" 14. Manually invoke \"reward\"")
```

Comment:
Just curious, what determines the ordering of options here? Appending new options at the end seems like it'd involve changing less code.

Reply:
I was hoping to have some kind of logical ordering, but yeah... not super clear here.  We can just add to the end from now on.